util/config/replace-paths.py

```diff
@@ -48,6 +48,14 @@
 
         origpath = m
         path = os.path.realpath(origpath)
+
+        if os.path.isfile(path):
+          # Leave out the filename, since we're only trying to replace
+          # directories. This prevents clang++ from being replaced
+          # with e.g. clang-6.0 when clang++ is a symbolic link.
+          origpath = os.path.dirname(path)
+          path = os.path.dirname(path)
```

Comment:
Shouldn't `path = os.path.realpath(origpath)` to get the canonicalized path name?  (That's a question, not a request.  I'm not sure I fully understand the original code.)

Reply:
yes it should thanks for spotting that